RAKE/RAKE.py

```diff
@@ -56,6 +65,15 @@ def split_sentences(text):
     return sentences
 
 
+def build_stop_word_regex_list(stop_word_list): #static handles built in files only
```

Comment:
You can remove this comment now.

Also you probably should change `build_stop_word_regex` to use this. It looks like it could be something like:

~~~
def build_stop_word_regex(stop_word_file_path):
  return build_stop_work_regex_list( load_stop_words(stop_word_file_path) )
~~~